src/parser/jobs/ast/index.tsx

```diff
@@ -34,8 +34,23 @@ export default new Meta({
 	},
 	contributors: [
 		{user: CONTRIBUTORS.SUSHIROU, role: ROLES.MAINTAINER},
+		{user: CONTRIBUTORS.CASUALSUPERMAN, role: ROLES.DEVELOPER},
 	],
 	changelog: [
+		{
+			date: new Date('2020-10-05'),
+			Changes: () => <>
+				Fixed <ActionLink {...ACTIONS.EARTHLY_STAR} /> drift calculation.
+			</>,
+			contributors: [CONTRIBUTORS.CASUALSUPERMAN],
+		},
+		{
+			date: new Date('2020-09-30'),
+			Changes: () => <>
+				Fixed bug where starting stance was always reported forgotten.
+			</>,
+			contributors: [CONTRIBUTORS.CASUALSUPERMAN],
+		},
```

Comment:
It's meant to be separate - I made this (and most of the other ones) as a draft and was going to publish it after #1023 got merged so I wasn't adding myself as a contributor in multiple PRs, getting conflicts later, etc.

Reply:
That's perfectly fine, but it's confusing from a 'hey, which PR made this change' perspective when there's two changes in the history list under a job associated only with a singular PR.

Reply:
Once you merge one, if @CasualSuperman back-merges master into the other branch, it will automatically remove the "duplicated" changeset from the other, in either order.

Reply:
Agreed, I wasn't sure what the best route was (didn't even know drafts were publicly listed tbh)

Reply:
Would it help if I abandoned the dependent PRs, and reopened them once the prior changes have been landed?